bandit/plugins/trojansource.py

```diff
@@ -24,8 +24,8 @@
 
 .. seealso::
 
- .. [1] https://trojansource.codes/
- .. [2] https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42574
+ - https://trojansource.codes/
```

Comment:
Are these supposed to be inside the see also block? If so they need a 3 space indent iirc

Reply:
Maybe GitHub mobile isn't rendering it properly locally since I checked https://bandit--1258.org.readthedocs.build/en/1258/plugins/b613_trojansource.html and it seems fine